template.yaml

```diff
@@ -170,6 +170,12 @@ Resources:
         Variables:
           TARGET_QUEUE: !GetAtt DeathEnrichmentQueue.QueueUrl
       AutoPublishAlias: DeathValidationFunction
+      VpcConfig:
+        SubnetIds:
+          - !Ref PrivateSubnet1
+          - !Ref PrivateSubnet2
+        SecurityGroupIds:
+          - !Ref LambdaSecurityGroup
```

Comment:
Is this something you can set on Global?

Reply:
Good idea, I'm gonna guess yes